### PR TITLE
MEV cleanup

### PR DESCRIPTION
**Pull request by nkryuchkov**
Adjust comments and docs

### CODE REVIEW

**Comment by nkryuchkov** [comment]
On hold until it's tested

**Comment by nkryuchkov** [comment]
Blocked by https://github.com/bloxapp/ssv/pull/952/commits/4b1b42302bd3504a85db07781e78e0e0bd5c9924

File: beacon/goclient/goclient.go
Comment (nkryuchkov) [inline beacon/goclient/goclient.go:128]: to be reverted

File: beacon/goclient/goclient.go
Comment (nkryuchkov) [inline beacon/goclient/goclient.go:145]: to be reverted

File: logging/fields/fields.go
Comment (nkryuchkov) [inline logging/fields/fields.go:303]: to be reverted

File: logging/fields/fields.go
Comment (nkryuchkov) [inline logging/fields/fields.go:55]: to be reverted

File: protocol/v2/blockchain/beacon/client.go
Comment (nkryuchkov) [inline protocol/v2/blockchain/beacon/client.go:67]: To be deleted

File: protocol/v2/ssv/runner/validator_registration.go
Comment (nkryuchkov) [inline protocol/v2/ssv/runner/validator_registration.go:24]: this should be done before merging

**Review by moshe-blox — APPROVED** [review]

**Pull request merged**
